docs/token-contracts/admin-lambda.md

```diff
@@ -0,0 +1,386 @@
+---
+id: admin-lambda
+title: Getting started with the AdminLambda contract
+sidebar_label: AdminLambda Quick Start
+---
+
+## Introduction
+
+The [AdminLambda](https://github.com/tqtezos/admin-lambda) contract
+accepts a `lambda` from the `SENDER` in its storage and executes it.
+That lambda may update the allowed-`SENDER` `address` in storage.
+
```

Comment:
Suggestion: use consistent term across the document (**administrator's address**)

> The AdminLambda contract
accepts a lambda from the **administrator's address** in its storage and executes it.
That lambda may update the **administrator's address** in storage.